core/src/main/scala/cr/pulsar/Producer.scala

```diff
@@ -121,19 +126,39 @@ object Producer {
   }
 
   /**
-    * It creates a simple [[Producer]] with a no-op logger.
+    * It creates a [[Producer]] with a no-op logger with shared key.
     *
     * Produced messages will not be logged.
     */
-  def create[
+  def withSharedKey[
```

Comment:
Should it be `withShardKey`?

Reply:
woops, was thinking about something else :)